specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/snapshot.json

```diff
@@ -561,6 +561,10 @@
           "type": "boolean",
           "description": "Whether a snapshot is incremental. Incremental snapshots on the same disk occupy less space than full snapshots and can be diffed."
         },
+        "incrementalSnapshotFamilyId": {
```

Comment:
is this a public property? I dont think it is

Reply:
It is